pkg/gce-pd-csi-driver/node.go

```diff
@@ -843,10 +841,12 @@ func (ns *GCENodeServer) GetVolumeLimits(ctx context.Context) (int64, error) {
 	gen4MachineTypesPrefix := []string{"c4a-", "c4-", "n4-"}
 	for _, gen4Prefix := range gen4MachineTypesPrefix {
 		if strings.HasPrefix(machineType, gen4Prefix) {
-			cpuString := machineType[strings.LastIndex(machineType, "-")+1:]
+			machineTypeSlice := strings.Split(machineType, "-")
+			cpuString := machineTypeSlice[2]
```

Comment:
This line will panic with runtime error if one day they decided to have a VM n4-micro.